src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -45,6 +47,16 @@ See the
 [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
+## Resources and Process Tags
+
+Jaeger sends information about the originating process once per exported batch.
```

Comment:
"per exported batch" - which batch? The batch that was sent from the processor to the exporter, or from the exporter to Jaeger backend?
* https://github.com/open-telemetry/opentelemetry-dotnet/blob/02ceb1a7642ae57adaee99ba393d05872b0ac47b/src/OpenTelemetry/BatchExportProcessor.cs#L222
* https://github.com/open-telemetry/opentelemetry-dotnet/blob/02ceb1a7642ae57adaee99ba393d05872b0ac47b/src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs#L67

Reply:
Re-worded to line to be less specific about export process. I believe from exporter to Jaeger backend.
![image](https://user-images.githubusercontent.com/29290481/111696371-797c7600-87f1-11eb-8a6b-dc242796e7c6.png)
inside JaegerExporter.cs